python/transformations/Sentiment-Analysis/library.json

```diff
@@ -34,7 +34,7 @@
     "DeploySettings":
     {
         "DeploymentType": "Service",
-        "CpuMillicores": 200,
+        "CpuMillicores": 1000,
```

Comment:
This is for sentiment analysis, OK as long as this is intentional.

Reply:
sorry i should have done on a separate PR but yes let's leave here this time